jix/urls.py

```diff
@@ -16,10 +16,13 @@
 from django.contrib import admin
 from django.urls import path, include
 from . import views
+from django.conf.urls import url
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', views.index, name="index"),
     path('events/', include('events.urls')),
-    path('reports/', include('report.urls'))
+    path('reports/', include('report.urls')),
+    url('', include('social_django.urls', namespace='social')),
```

Comment:
this can be refactor to `path('', include('social_django.urls', namespace='social'),`

Reply:
ok

Reply:
fixed it!